components/DisplayOwnCollection/index.js

```diff
@@ -0,0 +1,36 @@
+import { useState } from "react";
+
+const ownCollection = [
+  /* {
+    id: "1",
+    name: "Name des Comics",
+  },
+  {
+    id: "2",
+    name: "Name des Comics",
+  },
+  {
+    id: "3",
+    name: "Name des Comics",
+  }, */
```

Comment:
Remove lines of code which are commented out.

Reply:
removed